tests/http_server/test_local_http_server.py

```diff
@@ -0,0 +1,57 @@
+#  Copyright 2023 Google LLC.
+#  Copyright (c) Microsoft Corporation.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+
+from tests.test_helpers import execute_command
+
+
+async def get_content(websocket, context_id, url):
```

Comment:
add pydoc string:

```
"""Gets the body innerText content from the page with the given url."""
```

(just check whether "gets" or "get" based on the convention of the rest of our codebase)

Reply:
done